tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/query/XMLQueryExpressionTest.java

```diff
@@ -48,13 +48,13 @@ public void testSimpleQueryExprForXML() {
                 "<name>Sherlock Holmes</name><name>The Da Vinci Code</name>");
     }
 
-    @Test(description = "Test simple query expression for XMLs - #2")
+    @Test(groups = { "disableOnOldParser" }, description = "Test simple query expression for XMLs - #2")
     public void testSimpleQueryExprForXML2() {
         BValue[] returnValues = BRunUtil.invoke(result, "testSimpleQueryExprForXML2");
         Assert.assertNotNull(returnValues);
 
         Assert.assertEquals(returnValues[0].stringValue(),
-                "<book>the book</book>bit of text✂✅");
+                "<book>the book</book>bit of text\\u2702\\u2705");
```

Comment:
I think this is correct.
Spec [1] says:  Within a BacktickString, every character that is not part of an interpolation is interpreted as a literal character. 

[1] - https://ballerina.io/ballerina-spec/spec.html#section_6.6.1